src/Gui/Windows/CombinedCodeViewInteractor.cs

```diff
@@ -50,6 +50,9 @@ public class CombinedCodeViewInteractor : IWindowPane, ICommandTarget
 
         private DeclarationTextBox declarationTextBox;
 
+        private ImageSegment segment;
+        private bool hideProcedures;
```

Comment:
I prefer using "positive" property and field names whevener possible: prefer `showProcedures` to `hideProcedures` or `noShowProcedures`